src/CoreWlan/Enums.cs

```diff
@@ -116,4 +116,20 @@ public enum CWKeychainDomain : nuint_compat_int {
 		User = 1,
 		System = 2,
 	}
+
+	[Native]
+	public enum CWEventType : nuint_compat_int {
+		None = 0,
+		PowerDidChange = 1,
+		SSIDDidChange = 2,
+		BSSIDDidChange = 3,
+		CountryCodeDidChange = 4,
+		LinkDidChange = 5,
+		LinkQualityDidChange = 6,
+		ModeDidChange = 7,
+		ScanCacheUpdated = 8,
+		VirtualInterfaceStateChanged = 9,
+		RangingReportEvent = 10,
+		Unknown = int.MaxValue
```

Comment:
Updated the enum.
@rolfbjarne hope I got it right.
@chamons no worries, I understand that this is not a pet project :)
@dalexsoto thanks!